custom/icds_reports/management/commands/run_aggregation_query.py

```diff
@@ -85,6 +80,37 @@ class AggregationQuery(object):
     func = attr.ib()
 
 
+function_map = {}
+
+
+def setup_tasks():
```

Comment:
I refactored these functions out because it made it much easier to mock the tests.